zerver/lib/actions.py

```diff
@@ -3943,6 +3943,11 @@ def estimate_recent_invites(realm: Realm, *, days: int) -> int:
         return 0
     return recent_invites
 
+def invites_changed_event(realm: Realm) -> None:
+    recipient_for_event = {user.id for user in realm.get_admin_users()}
```

Comment:
let's call this admin_ids, to match the pattern in the rest of the file.
also a more efficient query is
admin_ids = list(realm.get_admin_users().values_list('id', flat=True))

You could also just inline it; so

Reply:
Changing the query to what you suggested, is showing the following mypy error.
`error: "Sequence[UserProfile]" has no attribute "values_list"`